pages/brand.js

```diff
@@ -262,6 +263,9 @@ const Page = ({ css }) => (
         Explore Hack Club Icons →
       </Button>
       <Heading variant="headline">UI components</Heading>
+      <Text variant="subtitle" sx={{ a: { variant: 'styles.a' }, mb: 3 }}>
+        <p>Want to make a Hack Club themed site? Use our pre-made CSS and UI components to hackify your site.</p>
+      </Text>
```

Comment:
```suggestion
      <Text variant="subtitle" sx={{ mb: 3 }} as="p">
        Want to make a Hack Club themed site? Use our pre-made CSS and UI components to hackify your site.
      </Text>
```

Reply:
I would have committed this suggestion but wanted to get everything in the commit. Does this look right? https://github.com/hackclub/site/pull/379/commits/451a1a550650a94416e21133fb156510d4d47b9a